packages/react-jss/test-utils/createDynamicStylesTests.js

```diff
@@ -277,6 +277,38 @@ export default ({createStyledComponent}) => {
       expect(passedProps.color).to.equal('rgb(255, 0, 0)')
       expect(passedProps.height).to.equal(20)
     })
+
```

Comment:
OK, I will give it a go. I gave up on that originally because it was taking me too long to figure out how to replicate what react-jss does to trigger the `this.deployed = false` line in `StyleSheet`.